src/logutil/logger.h

```diff
@@ -55,6 +55,8 @@ class Logger
   static void LogInfo(const string& message);
   static void LogWarn(const string& message);
   static void LogError(const string& message);
+  static string s;
```

Comment:
This is not required, and static objects must be initialized. That's why you are getting errors in the build.